api/api/utils/search_context.py

```diff
@@ -35,12 +35,10 @@ def build(
             # Use `identifier` rather than the document `id` due to
             # `id` instability between refreshes:
             # https://github.com/WordPress/openverse/issues/2306
-            # `identifier` is mapped as `text` which will match fuzzily.
-            # Use `identifier.keyword` to match _exactly_
-            # cf: https://github.com/WordPress/openverse/issues/2154
+            # `identifier` is a keyword field which will match exactly.
```

Comment:
This comment isn't _really_ necessary anymore now that there's only a single `identifier` field. I added it only to explain why the subfield was used rather than the root field, but that decision doesn't even exist in the code anymore after this.